hicommonwealth/commonwealth
# Removed `FLAG_NEW_ADMIN_ONBOARDING` + Cleanup and re-organized old comps

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6458

## Description of Changes
- Removed `FLAG_NEW_ADMIN_ONBOARDING`
- Removed old unused admin onboarding components
- Reorganized old reuseable admin onboarding components

## "How We Fixed It"
N/A

## Test Plan
In general, visit any admin-related page or area and verify nothing breaks. For specifics
- Visit any community where you are admin
- Visit `/manage/profile`, `/manage/integrations`, `/manage/topics`, `/manage/moderators`, verify that features on these pages work as expected and no UI element or action breaks (btn click, API call, scroll etc)
- On the same community (where you are admin), verify that sidebar section shows admin features and interacting with those features don't break anything

## Deployment Plan
N/A

## Other Considerations
We should cleanup `FLAG_NEW_ADMIN_ONBOARDING` from prod env.
